### PR TITLE
add take-last function

### PR DESCRIPTION
**Pull request by shmuelko**
add the possibility to use take-last function

### CODE REVIEW

File: src/practitest_firecracker/practitest.clj [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 ;; ===========================================================================
 ;; api version
-(def ^:const fc-version "1.4.0")
+(def ^:const fc-version "1.1.3")
```
Comment (stask) [inline src/practitest_firecracker/practitest.clj]: Why? version number should always increase, why did you decrease it?

File: src/practitest_firecracker/query_dsl.clj [hunk 2]
```diff
@@ -43,6 +43,11 @@
                                (throw
                                 (ex-info "Syntax error: 'take' must have two arguments"
                                          {:query query})))
+        'take-last           (if (= 2 (count args) 2)
+                               (take-last (parse-int (first args)) (last args))
+                               (throw
+                                (ex-info "Syntax error: 'take-last' must have one or two arguments"
```
Comment (stask) [inline src/practitest_firecracker/query_dsl.clj:51]: You don't currently support one argument in the code

**Review by stask — CHANGES_REQUESTED** [review]

File: src/practitest_firecracker/practitest.clj (hunk 1 above)
Comment (shmuelko) [reply src/practitest_firecracker/practitest.clj]: I think the old version was a mistake

File: src/practitest_firecracker/query_dsl.clj (hunk 2 above)
Comment (stask) [reply src/practitest_firecracker/query_dsl.clj:51]: This doesn't look good, maybe something like this:
```clojure
(case (count args)
   1 (take-last 1 (first args))
   2 (take-last (first args) (second args))
   (throw (ex-info "Syntax error: 'take-last' must have one or two arguments" {:query query})))
```

**Review by stask — CHANGES_REQUESTED** [review]

**Pull request closed without merging**